api/database/database.go

```diff
@@ -153,8 +154,13 @@ func SetupDatabase() (*gorm.DB, error) {
 	return db, nil
 }
 
-var database_models []interface{} = []interface{}{
+var database_models_1 []interface{} = []interface{}{
```

Comment:
I think the tests don't work since you make two database models and try to migrate them both. Why did you add a `database_models_1`?

Reply:
@viktorstrate the reason for this was due to some query size constraints atleast within mysql. I was going to investigate further to find a better solution. I will take a further look st this and hope its the cause I'd the failures